frontend/javascripts/oxalis/model/sagas/load_histogram_data_saga.js

```diff
@@ -46,21 +46,28 @@ export default function* loadHistogramData(): Saga<void> {
     const minimumInHistogramData = Math.min(...allMinValues);
     const maximumInHistogramData = Math.max(...allMaxValues);
     let newIntensityRange = [];
-    if (layerConfigurations[layerName]) {
+    const currentLayerConfig = layerConfigurations[layerName];
+    if (currentLayerConfig) {
       newIntensityRange = [
-        Math.max(layerConfigurations[layerName].intensityRange[0], minimumInHistogramData),
-        Math.min(layerConfigurations[layerName].intensityRange[1], maximumInHistogramData),
+        Math.max(
+          currentLayerConfig.intensityRange[0],
+          currentLayerConfig.min != null ? currentLayerConfig.min : minimumInHistogramData,
+        ),
```

Comment:
Can you add a comment which explains the intuition here? I think, I kind of got now, but I had think this through for a few minutes and I'm still not 100% sure. My take would be:
`intensityRange` is the range for which the histogram is plotted, right? By default, it's defined by the data type (e.g., uint16), but it can be configured per dataset layer.
That range serves as a boundary (i.e., the range cannot be increased by something else), but can still be narrowed down. If nothing else is provided, that down-narrowing is done by the sampled histogram. If the user specified a default min/max value, that is used instead.

Also interesting: intensityRange is more like a "theoretical" range which is not exceeded by the data (e.g., the microscope  did only emit values between x and y), whereas `min`/`max` is more like a recommended cut-off to get a good rendering?

Did I get this right? :laughing:

Reply:
> `intensityRange` is the range for which the histogram is plotted, right?

It is the exact opposite. The min/max is the range of the histogram (slider) while the `intensityRange` is the range to which the color values are clamped.

Is my new comment explaining the code good enough? I tried keeping it short.

Reply:
Ah, makes sense! Yes, should be good to go now :)